src/Serialization/serializers.jl

```diff
@@ -31,6 +31,11 @@ mutable struct SerializerState
   refs::Vector{UUID}
   io::IO
   key::Union{Symbol, Nothing}
+  type_attr_map::Function
+end
+
+function attrs_list(s::SerializerState, T::Type)
+  return get(s.type_attr_map, T, Dict{Type, Vector{Symbol}}())
```

Comment:
```suggestion
  return get(s.type_attr_map, T, Symbol[])
```
this of course needs to be the correct default...